tofu/geom/_core.py

```diff
@@ -3873,6 +3876,38 @@ def from_svg(
         verb=None,
         returnas=None,
     ):
+        """ Build a config from a svg file (Inkscape)
+
+        The svg shall have only:
+            - closed polygons (possibly inc. Bezier curves)
+            - an optional unique straight line (non-closed), for scaling
```

Comment:
"two points straight line"? or can it be more than 2 points?

Reply:
done